### DISCUSSION TITLE
Difference between model-best and model-last

### DISCUSSION THREAD

**Discussion by umayerr** [explosion/spaCy#12455] [Help: Best practices]
I took a spaCy blank model and trained that for `ner` with chemical data. At the end of the training process I found two different models (model-best and model-last) as output. I am wondering the difference between these two models? Which model should I use for entity recognition from the text? If I want to continue incremental training which model would be the perfect one to start?
Note: I am using spaCy v3.

**Comment by adrianeboyd** [comment]
Here's a closely related discussion: https://github.com/explosion/spaCy/discussions/9848#discussioncomment-1802297
